spec/features/accessibility/idv_pages_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 require 'axe/rspec'
 
-feature 'Accessibility on IDV pages', :js do
+feature 'Accessibility on IDV pages', :js, :accessibility do
```

Comment:
I like this tagging method. Might also make sense to include something in all of these accessibility specs that auto-includes this tag (and `:js`) so people don't need to remember to add this if/when future accessibility specs are added